newrelic-notify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex pipefail
+
+NEWRELIC_USER=gitlab
+
+API_KEY=$(< ~/.newrelic-api-key)
+
+# last 7 characters of the image value will be the hash
```

Comment:
now that we're grepping the tag out of the deployment, this comment is less applicable